docs/proposals/LightBox/README.md

```diff
@@ -0,0 +1,72 @@
+# LightBox
+
+A `<Lightbox />` component will be used to display an image to the user. This
+component will incorporate image controls (ie. Zoom, next/prev, exit).
+
+This LighBox component will follow the industry established design patterns, but
+will use Jobber color palettes and fonts.
+
+As this a common industry pattern we will seek to leverage existing React
+libraries. Specifically `frontend-collective/react-image-lightbox`.
+
+## Interface
+
+```
+<LightBox
+    mainSrc="image_url"
+    prevSrc="image_to_the_left_url"
+    nextSrc="image_to_the_right_url"
+    onCloseRequest=function_to_close
+/>
+```
+
+## Props Table
+
+| name     | type   | default  | description                                              |
+| -------- | ------ | -------- | -------------------------------------------------------- |
+| image    | String | -        | The avatar image to be use                               |
+| name     | String | -        | A users name to be used for accessibility                |
+| initials | String | -        | Initials to show if the image is null                    |
+| color    | String | greyBlue | The background and border color that represents the user |
+| size     | String | small    | Size of the avatar - 'small'                             | 'large' |
+
+## Display
+
+An `<Avatar />` component would have a heiarchy of display options
```

Comment:
Not sure if we need this file. It also mostly talks about the `Avatar`. 😄

Reply:
Ah crap, forgot to delete this.